### PR TITLE
Added delay_constant to sort-tags.xsl

### PR DESCRIPTION
**Pull request by glatosinski**
Signed-off-by: Grzegorz Latosinski <glatosinski@antmicro.com>

### CODE REVIEW

**Review by mithro — COMMENTED** [review]
Can you add a test?

**Comment by glatosinski** [comment]
I added a simple test that verifies sorting of `input` and `delay_constant` tags

File: tests/convert_and_merge_composable_tests/tag-sorting.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='utf-8'?>
```
Comment (mithro) [inline tests/convert_and_merge_composable_tests/tag-sorting.xml:1]: You probably want to add multiple out ports to test that it is sorting via outport first and then inport?

**Review by mithro — APPROVED** [review]
One minor comment.

File: tests/convert_and_merge_composable_tests/tag-sorting.xml (hunk 1 above)
Comment (glatosinski) [reply tests/convert_and_merge_composable_tests/tag-sorting.xml:1]: I added additional output to make this check

**Review by mithro — APPROVED** [review]

**Comment by mithro** [comment]
LGTM! Merged!

**Pull request merged**